pkg/sql/delegate/show_database_indexes.go

```diff
@@ -28,7 +36,8 @@ SELECT
 	column_name,
 	direction,
 	storing::BOOL,
-	implicit::BOOL`
+	implicit::BOOL,
+  is_visible::BOOL AS visible`
```

Comment:
nit: use tabs not spaces for consistency

(alternatively, replace the other tabs with spaces)